webmails/snuffleupagus.rules

```diff
@@ -84,6 +84,7 @@ sp.disable_function.function("ini_set").param("option").value("include_path").dr
 sp.disable_function.function("ini_set").param("option").value("open_basedir").drop();
 
 # Detect some backdoors via environment recon
+sp.disable_function.function("ini_get").filename_r("/var/www/roundcube/vendor/guzzlehttp/guzzle/src/functions\.php$").param("option").value("allow_url_fopen").allow();
```

Comment:
Please use filename() instead; no need for the regexp engine here.

Reply:
I did first, but then noticed on line 101 you used filename_r and wanted to keep consistent: https://github.com/Mailu/Mailu/blob/d0061b95e8dda06ee79d2337b15cbbb1d123ffd9/webmails/snuffleupagus.rules#L101

Any reason for this?

Reply:
That has an actual regexp; in snappymail the path changes for each version.

Reply:
@nextgens sorry but I don't see the version check in the following string: `/app/libraries/snappymail/pgp/gpg\.php$`
Am I missing something or is it a mistake?

Reply:
The version is on the left side of the path... that's why this is a regexp.

The full path is:
``/var/www/snappymail/snappymail/v/2.21.3/app/libraries/snappymail/pgp/gpg.php``

Reply:
If you want to change it to be more specific, be my guest :)

``^/var/www/snappymail/snappymail/v/\d+\.\d+\.\d+/app/libraries/snappymail/pgp/gpg\.php$``

Reply:
gotcha now, thanks :) I'll make it more specific then so it's more clear ;)